src/components/10-atoms/checkbox/index.scss

```diff
@@ -1,4 +1,5 @@
-$checkbox-dimension: 18px;
+$checkbox-dimension: 20px;
```

Comment:
You changed the size of the checkbox? Is that change approved by a designer?

Reply:
Jep it was requested